streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/rule/sql/SqlScript.java

```diff
@@ -142,7 +154,7 @@ public StreamlineEvent apply(Values values) {
                         return valuesConverter.convert(values, inputEvent);
                     }
                 });
-            } else {
+            } else if (result instanceof StreamlineEvent){
```

Comment:
And is it valid to return empty list? If we expect either valuesConverter is provided or result is a StreamlineEvent instance, let's throw exception for other cases.